openssl/uwp.mk

```diff
@@ -28,16 +28,16 @@ PROXY_INTF_SOURCEFILES = ..\core\pubnub_proxy.c ..\core\pubnub_proxy_core.c ..\c
 PROXY_INTF_OBJFILES = pubnub_proxy.obj pubnub_proxy_core.obj pbhttp_digest.obj pbntlm_core.obj pbntlm_packer_sspi.obj pubnub_set_proxy_from_system_windows.obj 
 !endif
 
-!ifndef USE_GRANT_TOKEN
-USE_GRANT_TOKEN = 1
+!ifndef USE_PAM_V3
+USE_PAM_V3 = 1
 !endif
 
-!if $(USE_GRANT_TOKEN)
-GRANT_TOKEN_SOURCEFILES = ..\core\pbcc_grant_token_api.c ..\core\pubnub_grant_token_api.c ..\lib\cbor\cborparser.c ..\lib\cbor\cborerrorstrings.c ..\lib\cbor\cborparser_dup_string.c  
-GRANT_TOKEN_OBJFILES = pbcc_grant_token_api.obj pubnub_grant_token_api.obj cborparser.obj cborerrorstrings.obj cborparser_dup_string.obj  
+!if $(USE_PAM_V3)
+PAM_V3_SOURCEFILES = ..\core\pbcc_grant_token_api.c ..\core\pubnub_grant_token_api.c ..\lib\cbor\cborparser.c ..\lib\cbor\cborerrorstrings.c ..\lib\cbor\cborparser_dup_string.c ../core/pubnub_revoke_token.c ../core/pbcc_revoke_token.c
```

Comment:
for revoke files, we need to fix the paths. It should have backward slash, not forward slash.

Reply:
👌